snapcraft/internal/repo/_platform.py

```diff
@@ -26,6 +26,7 @@
     '"elementary"',
     'debian',
     'neon',
+    'LinuxMint',
```

Comment:
long term it would be good if snapcraft could switch to a function that uses /etc/os-release so we are all on the same page (i dont think snapd uses lsb stuff anywhere, since there is no guarantee it is even installed)